doc/tutorials/core/univ_intrin/univ_intrin.markdown

```diff
@@ -244,10 +244,12 @@ The universal intrinsics set provides element wise binary and unary operations.
 
 ## Demonstration
 In the following section, we will vectorize a simple convolution function for single channel and compare the results to a scalar implementation.
-@note The implementation can be improved for faster results. However, further improvements may make the tutorial complicated.
+@note Not all algorithms are improved by vectorization. In fact, in certain cases, the compiler may make improvements to the code automatically, thus producing faster results for scalar implementations.
```

Comment:
`by vectorization` -> `by manual vectorization`
'make improvements to` -> `autovectorize`
to make cleaner the reason why vectorization could yield no benefit